app/models/User.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
+      name: { type: DataTypes.STRING, allowNull: false },
+      lastName: { type: DataTypes.STRING, allowNull: false, field: 'last_name' },
+      email: { type: DataTypes.STRING, allowNull: false, unique: true },
+      password: { type: DataTypes.STRING, allowNull: false },
+      created_at: { type: DataTypes.DATE },
+      updated_at: { type: DataTypes.DATE }
```

Comment:
You don't need to add this, but if you want to keep them, they should be in `camelCase` in the model. And remember the file name, it should be in lowercase